Templates/Kotlin/InputWrapper.stencil

```diff
@@ -3,19 +3,26 @@ import javax.annotation.Generated
 
 @Generated("com.shopify.syrup")
 class InputWrapper<T>() {
-    private var value: T? = null
+    var value: T? = null
+        set(value) {
+            defined = true
+            field = value
+        }
+
     private var defined = false
 
     constructor(t: T?): this() {
         this.value = t
         this.defined = true
     }
 
+    @Deprecated("Using the setter of value directly will also set `defined` equal to true")
```

Comment:
Nice that we're providing a migration path but unsure we should bother since we haven't made an "official" release yet. Personally, I'd lean towards not having any API contracts to maintain yet.